ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -662,6 +662,10 @@ export const getMhrRegistrationHomeOwners = (state: StateIF): MhrRegistrationHom
   return owners
 }
 
+export const getMhrRegistrationDocumentId = (state: StateIF): any => {
```

Comment:
We could probably declare a `string` type return, since we know that for sure here

Reply:
Added return type, thanks!